src/stcal/jump/twopoint_difference.py

```diff
@@ -1,19 +1,21 @@
 import logging
 import numpy as np
-
+import astropy.stats as stats
+import warnings
 log = logging.getLogger(__name__)
 log.setLevel(logging.DEBUG)
 
 
-def find_crs(dataa, group_dq, read_noise, rejection_thresh,
+def find_crs(dataa, group_dq, read_noise, normal_rej_thresh,
              two_diff_rej_thresh, three_diff_rej_thresh, nframes,
              flag_4_neighbors, max_jump_to_flag_neighbors,
              min_jump_to_flag_neighbors, dqflags,
              after_jump_flag_e1=0.0,
              after_jump_flag_n1=0,
              after_jump_flag_e2=0.0,
              after_jump_flag_n2=0,
-             copy_arrs=True):
+             copy_arrs=True, minimum_groups=3, minimum_sigclip_groups=100,
+             only_use_ints=True):
```

Comment:
Need to add the 3 new params to the param list doc string below.

Reply:
Done